model-optimizer/extensions/front/mxnet/gluoncv_ssd_anchors.py

```diff
@@ -0,0 +1,141 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import numpy as np
+
+from typing import Dict
+
+from extensions.front.mxnet.ssd_detection_output_replacer import SsdPatternDetectionOutputReplacer
+from extensions.ops.elementwise import Div, Add, Sub
+from extensions.ops.split import Split
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.front.tf.graph_utils import create_op_node_with_second_input
+from mo.graph.graph import Graph, Node
+from mo.graph.port import Port
+from mo.middle.passes.convert_data_type import data_type_str_to_np
+from mo.ops.concat import Concat
+from mo.ops.reshape import Reshape
+
+
+def get_coords(graph: Graph, value: Node, div_value_port: Port, add_value_port: Port):
+    dtype = data_type_str_to_np(graph.graph['cmd_params'].data_type)
+    _min = Sub(graph, dict(name=value.name + '/Sub')).create_node()
+    div = create_op_node_with_second_input(graph, Div, np.array([2], dtype=dtype), op_attrs=dict(name=value.name + '/Div'))
+    div.in_port(0).connect(div_value_port)
+    _min.in_port(0).connect(add_value_port)
+    _min.in_port(1).connect(div.out_port(0))
+
+    _max = Add(graph, dict(name=value.name + '/Add')).create_node()
+    _max.in_port(0).connect(div.out_port(0))
+    _max.in_port(1).connect(add_value_port)
+
+    return _min, _max
+
+
+class SsdAnchorReshape(FrontReplacementPattern):
+    enabled = True
+    graph_condition = [lambda graph: graph.graph['fw'] == 'mxnet' and graph.graph['cmd_params'].enable_ssd_gluoncv]
```

Comment:
Check for `graph: graph.graph['fw'] == 'mxnet'` is redundant because this transformation is in the MxNet specific directory